values/binary_test.go

```diff
@@ -552,8 +556,11 @@ func TestBinaryOperator(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-
-			if want, got := Value(tt.want), fn(left, right); !ValueEqual(want, got) {
+			want := Value(tt.want)
+			got, err := fn(left, right)
+			if err != nil {
+				fmt.Printf("Dividing by zero")
```

Comment:
I'm not positive if this will accurately account for the new test cases on lines 75 and 88. Without this if statement, I get a panic (seg fault).

Reply:
It would be good to account for a possible error produced by the test case by adding an `err` field to the struct slice being looped over at line 26.  Then you can change the logic here to look for an error coming from `fn` if `tc.err` is not nil.

An example of a test written this way is here: `execute/executor_test.go`, but there's several like this.